src/CMakeLists.txt

```diff
@@ -234,7 +234,7 @@ install(FILES
     ${CMAKE_BINARY_DIR}/exiv2lib_export.h
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/exiv2)
 
-install(EXPORT exiv2Config DESTINATION "share/exiv2/cmake")
+install(EXPORT exiv2Config DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/exiv2")
```

Comment:
I have tested this with the [exiv2Consumer](https://github.com/piponazo/exiv2Consumer) project and it works.

Just as a side note: the previous path was also correct. `find_package` try to search in different locations and both (the previous path and the new one work):
https://cmake.org/cmake/help/latest/command/find_package.html?highlight=find_package#search-procedure